styles/fonts.css

```diff
@@ -30,3 +30,18 @@
   src: url('../fonts/Lato-BoldItalic.ttf') format('truetype');
 }
 
+@font-face {
+  font-family: "Font Awesome\ 6 Brands";
+  font-style: normal;
+  font-weight: 400;
+  font-display: block;
+  src: url('../fonts/fa-brands-400.woff2') format("woff2");
+}
+
+@font-face {
+  font-family: "Font Awesome\ 6 Free";
+  font-style: normal;
+  font-weight: 900;
+  font-display: block;
+  src: url('../fonts/fa-solid-900.woff2') format("woff2");
+}
```

Comment:
Only a comment: Not sure it's more performant to export the ones we need and use the EDS icon system. The FA font is probably more versatile though